provisioning_templates/finish/preseed_default_finish.erb

```diff
@@ -53,4 +53,5 @@ oses:
 <% end -%>
 
 <%= snippet 'preseed_networking_setup' %>
+<%= snippet 'efibootmgr_netboot' %>
```

Comment:
was this tested with Debian or Ubuntu?

Reply:
It was not, however this is opt-in only. No changes are expected if no host param is provided.